tests/mongo-go-driver.yml

```diff
@@ -119,3 +212,372 @@ results:
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-crud.json
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-crud.json/Aggregate_with_$listLocalSessions
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-crud.json/BulkWrite_with_mixed_ordered_operations
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-crud.json/InsertMany_continue-on-error_behavior_with_unordered_(duplicate_key_in_requests)
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-crud.json/ReplaceOne_prohibits_atomic_modifiers
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-gridfs.json
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-gridfs.json/Delete_when_length_is_10
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-gridfs.json/Download_when_an_intermediate_chunk_is_missing
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-gridfs.json/Download_when_files_entry_does_not_exist
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-gridfs.json/Download_when_there_are_three_chunks
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-gridfs.json/Upload_when_length_is_5
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-retryable-reads.json/Aggregate_succeeds_after_InterruptedAtShutdown
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-retryable-reads.json/Find_fails_on_first_attempt
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-retryable-reads.json/Find_succeeds_on_second_attempt
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-retryable-reads.json/ListDatabases_succeeds_on_second_attempt
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-sessions.json
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-sessions.json/Server_supports_explicit_sessions
+      - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec/unified-test-format/valid-pass/poc-sessions.json/Server_supports_implicit_sessions
```

Comment:
There's still a lot of mess in the pass section. I'm not sure if I'm doing everything correctly, so I will be happy if anyone take a look and tell me if there is any implemented way to reduce amount of lines here or should I make another PR which will allow us to match testnames with regex?

For now I tried to pass "/" at the end of the testname to match all tests under it but it didn't work as I expected.